### PR TITLE
Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by itsJakov**
This should work, although i'm not sure this didn't break CocoaPods support.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+let package = Package(
+    name: "Ecno",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "Ecno",
+            targets: ["Ecno"]),
+    ],
+    dependencies: [
```
Comment (mats-claassen) [inline Package.swift]: You don't need to specify the dependencies key.

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Ecno",
```
Comment (mats-claassen) [inline Package.swift]: Please add a `platform` key that specifies the minimum iOS version supported

**Review by mats-claassen — CHANGES_REQUESTED** [review]

**Comment by mats-claassen** [comment]
Hi @itsJakov Thanks for contributing. I wonder why there are so many changes to the project file. And why you uploaded two new Info.plist files. Supporting Package Manager should only require the Package.Swift file.

**Comment by itsJakov** [comment]
Adding the `Package.swift` file manually works much better than `swift package init — type library` and `swift package generate-xcodeproj`.

**Review by mats-claassen — APPROVED** [review]

**Pull request merged**